### PR TITLE
[Docs][small] Updated e2e dapp tutorial's Docs

### PR DESCRIPTION
**Pull request by satyvm**
Basic code error in the tutorial's code snippet (an extra semicolon).

### CODE REVIEW

**Comment by satyvm** [comment]
Here is the code written with the updated docs, https://github.com/satyvm/aptostodolist. And here is the working website, https://aptostodolist.vercel.app/ to confirm the errors.

File: developer-docs-site/docs/tutorials/build-e2e-dapp/6-handle-tasks.md [hunk 1]
```diff
@@ -429,6 +429,8 @@ We make sure there is an account connected, set the transaction in progress stat
 ...
 ```
 
-Try it! Check a task’s checkbox, approve the transaction and see the task marked as completed.
+4. You might be getting an error for missing components. To resolve, update the import, ```import { Layout, Row, Col, Button, Spin } from "antd";``` to ```import { Layout, Row, Col, Button, Spin, List, Checkbox, Input } from "antd";```.
```
Comment (0xmaayan) [inline developer-docs-site/docs/tutorials/build-e2e-dapp/6-handle-tasks.md:432]: Thanks! We probably need to ask to update the import statement on the step we use the new components

**Review by 0xmaayan — APPROVED** [review]
Thanks!

**Comment by 0xmaayan** [comment]
fixed on https://github.com/aptos-labs/developer-docs/pull/17

**Pull request closed without merging**